access/email/README.md

```diff
@@ -15,11 +21,34 @@ $ cd teleport-access-email
 $ ./install
 ```
 
-### Teleport User and Role
+### Docker Image
+```bash
+$ docker pull quay.io/gravitational/access-plugin-email:9.0.2
+```
+
+```bash
+$ docker run quay.io/gravitational/access-plugin-email:9.0.2 version
+teleport-email v9.0.2 git:teleport-email-v9.0.2-0-g9e149895 go1.17.8
+```
+
+For a list of available tags, visit [https://quay.io/](https://quay.io/repository/gravitational/access-plugin-email?tab=tags)
+
+### Building from source
+
+To build the plugin from source you need Go >= 1.16 and `make`.
```

Comment:
I'd avoid duplicating the go version in the readme. This feels like it will fall out of date.  Instead, maybe link to where the go version is set?

Reply:
Fixed in https://github.com/gravitational/teleport-plugins/pull/473/commits/1124b8c56e4e0056573473df3a084931c75744d1